db/migrate/20231219122845_change_cost_column_type.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeCostColumnType < ActiveRecord::Migration[7.1]
+  def change
+    rename_column :costs, :type, :cost_type
+  end
+end
```

Comment:
**nit** either way, but could  combine this migration with the on above as they are part of the same PR. use of the term `type` (rather than cost_type) could, in any event provide a simple means to use single table inheritance pattern in future if deemed necessary. but i don't know enough about the intent and possible extension here so either way on the naming too.

Reply:
Using 'type' got me into too many binds when trying to code this because it's a reserved column name

Reply:
you can still use `type` as a columkn name and avoid the STI issues if change this value

https://apidock.com/rails/ActiveRecord/ModelSchema/ClassMethods/inheritance_column